deploy/docker/notebook/base/base.yaml

```diff
@@ -421,26 +425,92 @@ addfiles:
           if "NB_UMASK" in os.environ:
               os.umask(int(os.environ["NB_UMASK"], 8))
 
-          def is_inside_path(path_to_test, path):
-              return os.path.samefile(os.path.commonpath([path, path_to_test]), path)
-        
-          # Configure file post-save hook
-          def script_post_save(model, os_path, contents_manager, **kwargs):
+          # Configure file post-save, post-rename, and post-delete hooks
+          
+          #### Hook Class ####
+
+          class hookVariables():
+              POLUS_NOTEBOOKS_HUB_API = os.environ["POLUS_NOTEBOOKS_HUB_API"]
               work_path = '/home/jovyan/work/'
               shared_path = '/home/jovyan/shared/notebooks'
```

Comment:
Why not put these variables to the `HookFileContentsManager` constructor? Two arguments in favor of that:
- `hookVariables` is not used anywhere outside of `HookFileContentsManager` class
- We don't want to create identical objects of `hookVariables` class every time notebook is saved/renamed/deleted (possible memory leak), which is currently happening in this PR